src/presentation/components/ItemCards.tsx

```diff
@@ -30,14 +30,25 @@ const ItemCards: React.FC = () => {
 
   useEffect(() => {
     //Resolves promise to set the data from JSON into the custom hook.
-    apiCallFunc().then(res => setData(res.items));
+    fetchItems()
```

Comment:
instead of calling `fetchItems` directly here, please call it in `useItemFetch`. Then, call `setData` directly in `useItemFetch`, not here.

We want to move all logic out of this file and into the application layer. 

Ideally there should be:
`const { data, showMore, showMoreItems, listItems } = useItemFetch(); `
and then you `listItems` directly very cleanly without setting data in this file